src/components/molecules/PieChartView.tsx

```diff
@@ -125,7 +126,6 @@ export const PieChartView: FC<IProps> = ({
           nameKey={xLabel}
           innerRadius={innerRadius}
           outerRadius={outerRadius}
-          minAngle={15}
```

Comment:
@yuvalbl - The size of the pie angles doesn't match the percentage it represents (for example the red 25% takes more than a quarter in the pie itself)